CHANGELOG.md

```diff
@@ -1,266 +1,210 @@
 -   [Pull requests merged in this release](#pull-requests-merged-in-this-release)
 -   [Known issues](#known-issues)
 -   [Upgrading from a previous version](#upgrading-from-a-previous-version)
 
-## What's new in 7.0.0
+## What's new in 8.0.0
 
-PWA Studio 7.0.0 contains new features, refactors, and various improvements.
+PWA Studio 8.0.0 contains new features, refactors, and various improvements.
 
-### Extensibility framework
+### Improved performance
 
-This release improves on the extensibility framework introduced in version 6.0.0.
-It introduces new extension points for the Buildpack, Peregrine, and Venia UI library components.
+A lot of work has been done in this release to improve storefront performance.
+This means that any project using the latest PWA Studio components will benefit from these updates.
 
-Developers can use these extension points to extend their storefront project without duplicating and maintaining PWA-Studio code.
+One of the bigger changes is the migration to [Apollo Client 3.0][].
+This version of the Apollo Client provides better cache controls and better network fetching performance in general.
 
-As of 7.0.0, PWA-Studio contains the following extension points:
+This release also includes various refactors and improvements on the GraphQL queries themselves to reduce the API request times from Magento.
 
-#### Venia UI extension points
+[apollo client 3.0]: https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/
 
-Venia UI extension points are declared in [`venia-ui-declare.js`][].
+### Complete cart and checkout experience
 
-| Target name            | Description                                          |
-| ---------------------- | ---------------------------------------------------- |
-| `richContentRenderers` | Add custom rich content renderers to your storefront |
-| `routes`               | Add or modify storefront-specific routes             |
+This release finishes the full page cart and checkout features introduced in previous releases.
+The complete cart and checkout workflow is based on research made by members of the UX team.
 
-#### Peregrine extension points
+Storefront developers can use this streamlined process as implemented or they can use the different components to customize their own cart and checkout workflow.
 
-Peregrine extensions points are declared in [`peregrine-declare.js`][].
+#### Shopping Bag feature
 
-| Target name | Description                                                                   |
-| ----------- | ----------------------------------------------------------------------------- |
-| `talons`    | Intercept specific Peregrine talons and wrap them with your custom components |
+In addition to the complete full page cart and checkout experience, this release introduces a new Mini-Cart/Shopping Bag feature.
 
-Wrappable talons:
+This feature appears as a floating modal that appears when you click on the shopping bag icon.
```

Comment:
Should read: "This feature is a floating modal that appears when you click on the shopping bag icon."